relaton/models/strings.py

```diff
@@ -11,7 +11,7 @@ class FormattedContent:
     """
     Relaton’s formatted string.
     """
-    content: str
+    content: Optional[str] = None
```

Comment:
I see why. This comment: https://github.com/relaton/relaton-py/pull/40#issuecomment-1230118553

Reply:
@stefanomunarini (and @andrew2net)

According to what I see [in RNC](https://github.com/relaton/relaton-models/blob/0d06e95fb2eabfa30d95d274f4055e7b7cb3c943/grammars/biblio.rnc#L56-L83), formatted/localized string must have content.

But I understand that we have a long conflict between a desire to make YAML readable vs. easily machine-parseable.

@stefanomunarini let’s keep it required?  If we make it optional here to make Forename work, it will affect all other formatted strings. Unless someone can show why formatted string with only `format` and no `content` is necessary, we should keep content required. 

To make forenames work specifically, we can normalize missing content to an empty string before passing data to Relaton.